tests/test_routes.py

```diff
@@ -173,14 +171,34 @@ def test_get_wishlist_list(self):
                     "id": wishlist_id,
                     "customer_id": customer_id,
                     "wishlist_name": wishlist_name,
-                    "created_date": created_date
+                    "created_date": created_date,
                 }
             )
-        resp = self.client.get(
-            BASE_URL,
-            content_type="application/json"
-        )
+        resp = self.client.get(BASE_URL, content_type="application/json")
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         data = resp.get_json()
         self.assertEqual(len(data), 5)
         self.assertEqual(data, wishlist_array)
+
+    ######################################################################
+    #  WishlistItem Test Cases
+    ######################################################################
+
+    def test_get_wishlist_item_list(self):
+        """It should Get a list of items"""
+        # add two items to wishlist
+        wishlist = WishlistFactory()
+        items = WishlistItemFactory.create_batch(2)
+
+        wishlist.items.extend(items)
```

Comment:
Interesting python methods called here. 
It might be confusing for others down the line, but if it works.